pages/packages/maven.md

```diff
@@ -48,18 +53,85 @@ The following steps describe the process above:
     // distributionManagement: 'maven deploy' pushes to this repository
     <distributionManagement>
       <repository>
-        <id>org-slug-java-maven-package-repository-name</id>
-        <url>https://buildkitepackages.com/{org.slug}/{java.maven.package.repository.name}</url>
+        <id>org-slug-java-package-repository-name</id>
+        <url>https://buildkitepackages.com/{org.slug}/{java.package.repository.name}</url>
       </repository>
       <snapshotRepository>
-        <id>org-slug-java-maven-package-repository-name</id>
-        <url>https://buildkitepackages.com/{org.slug}/{java.maven.package.repository.name}</url>
+        <id>org-slug-java-package-repository-name</id>
+        <url>https://buildkitepackages.com/{org.slug}/{java.package.repository.name}</url>
       </snapshotRepository>
     </distributionManagement>
     ```
 
+    where:
+    * `org-slug-java-package-repository-name` is the ID of your Java package repository (above).
+
+    <%= render_markdown partial: 'packages/org_slug' %>
+
+    <%= render_markdown partial: 'packages/java_package_repository_name' %>
+
 1. Publish your package:
 
     ```bash
     mvn deploy
     ```
+
+## Access a package's details
+
+<%= render_markdown partial: 'packages/access_java_package_details_page' %>
+
+<%= render_markdown partial: 'packages/package_details_page_sections' %>
+
+### Downloading a package
+
+A Java (Jar) package can be downloaded from the package's details page.
+
+To download a package:
+
+1. [Access the package's details](#access-a-packages-details).
+1. Select _Download_.
+
+### Installing a package
+
+1. [Access the package's details](#access-a-packages-details).
+1. Ensure the _Installation_ > _Installation instructions_ section is displayed.
+1. Copy the code snippet and paste this into the `pom.xml` file (under the `project` XML tag) to install this package.
+
+The package code snippet is based on this format:
+
+```xml
+// Add this to 'pom.xml' under 'project' xml tag
+<repositories>
+  <repository>
+    <id>org-slug-java-package-repository-name</id>
+    <url>https://buildkitepackages.com/priv/{api.token}/{org.slug}/{java.package.repository.name}/maven2/</url>
```

Comment:
Same here:

> As mentioned, I don't think we should be supplying `/priv/{token}` urls, but we need to fix the in product.

Reply:
Again, thanks @sj26 , I've removed this and will await any further instructions required on how the API read token is configured for installing Maven packages.